src/Api/Dockerfile

```
@@ -0,0 +1,43 @@
+
+# 🚀 Publish
+FROM build AS publish
+RUN dotnet publish "OPS.Api.csproj" -c Release -o /app/publish
+
+# 🎯 Final stage
+FROM mcr.microsoft.com/dotnet/aspnet:9.0
+WORKDIR /app
+
+# 🔐 Create a non-root user and group
+RUN addgroup --system --gid 1001 opsapi
+RUN adduser --system --uid 1001 opsuser
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)